cmd/nirvana/project/init.go

```diff
@@ -214,6 +221,7 @@ func (o *initOptions) directories(project string) []string {
 }
 
 func (o *initOptions) templates(proj string) map[string]string {
+	pkgConfPath, templateGopkg := o.templatePackageManager()
```

Comment:
Variable `templateGopkg` is not good. It may be a template of `mod`.

Reply:
change it to` templatePkgmgr` ? Or, rename `templatePakcageManager` to other name, let `templateGopkg` be `templatePackageManager`.

Reply:
`templatePkgmgr` and `templatePackageManager` are ok.

Reply:
Ok.